code/style.css

```diff
@@ -0,0 +1,205 @@
+body {
+    font-family: 'Montserrat', sans-serif;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    margin: 0px;
+
+}
+
+.bot-container {
+    display: flex;
+    align-items: center;
+    width: 80%;
+    position: absolute; 
+    top: 45%; 
+    left: 50%; 
+    margin-right: -50%; 
+    transform: translate(-50%, -50%)
+
+}
+
+.bot-content {
+    width: 60%;
+}
+
+.bot-container img {
+    width: 100%;
+}
+
+.bot-pic {
+    width: 40%;
+}
+
+.bar-container {
+    background-color: #f4f4f4;
+    border-radius: 10px;
+    display: none;
+}
+
+.bar-progress {
+    background-color: #185272;
+    width: 0%;
+    height: 15px;
+    border-radius: 10px;
+}
+
+
+.bot-text {
+    font-size: 50px;
+}
+
+.bot-content h2 {
+    font-weight: 400;
+    font-size: 30px;
+    color: #185272;
+}
+
+.pizza-container {
+    display: none;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.quantity-container {
+    display: none;
+}
+
+@keyframes bounce {
+	0%, 40%, 80%, 100% {
+		-webkit-transform: translateY(0);
+		transform: translateY(0);
+	}
+
+	20% {
+		-webkit-transform: translateY(-20px);
+		transform: translateY(-20px);
+	}
+
+	60% {
+		-webkit-transform: translateY(-10px);
+        transform: translateY(-10px);
+  }
+
+}
+
+.pizza img {
+    width: 100%;
+    border-radius: 10px; 
+    cursor: pointer;
+
+}
+
+.pizza img:hover {
+    animation: bounce 1s;
+
+
+}
+.pizza {
+    width: 30%;
+    box-sizing: border-box;
+    padding: 15px;
+}
+
+
+
+.pizza h2 {
+    font-weight: 400;
+    font-size: 20px;
+    text-align: center;
+}
+
+.quantity {
+    width: 100px;
+    height: 35px;
+    border-radius: 10px;
+    border: solid #185272;
+    padding-left: 10px;
+    margin-right: 20px;
+
+
+}
+
+.start-btn {
+    width: 200px;
+    height: 40px;
+    background-color: #185272;
+    border: none;
+    color: #fff;
+    border-radius: 10px;
+    font-size: 15px;
+    font-weight: 700;
+    letter-spacing: 1.5px;
+    cursor: pointer;
+}
+
+.start-btn:hover {
+    background-color: #e1f4f0;
+    color:  #185272;
+}
+
+footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    text-align: center;
+    font-size: 15px;
+    color: #185272;
+    
+}
```

Comment:
The short answer to your footer question is that you would have to put a height on the `footer` here so you can decide a matching bottom margin on the your `body` .

When changing an element to `position: absolute`  it loses the ability to be positioned relative to other elements